docs/fsharp/introduction-to-functional-programming/index.md

```diff
@@ -1,8 +1,221 @@
 ---
-redirect_url: /dotnet/fsharp/introduction-to-functional-programming/functions-as-first-class-values
+title: Introduction to Functional Programming in F#
+description: Learn the fundamentals of functional programming in F#.
+ms.date: 10/15/2018
 ---
 
 # Introduction to Functional Programming in F# #
 
-> [!NOTE]
-This is still in-progress.
\ No newline at end of file
+Functional programming is a style of programming that emphasizes the use of functions and immutable data. Typed functional programming is when functional programming is combined with static types, such as with F#. In general, the following concepts are emphasized in functional programming:
+
+* Functions over objects
```

Comment:
I like this list as a set of goals. You did a great job of explaining immutable values over variables. I didn't find any comparison of functions and objects, or declarative over imperative. I mentioned some suggestions to make the comparison of expressions and suggestions stronger.  

If your intended audience is developers familiar with OO or procedural languages, I'd make those comparisons more clear.